platform/admin/admin-configuration-configupdater/src/main/java/org/codice/ddf/admin/configuration/ConfigurationUpdater.java

```diff
@@ -251,8 +251,7 @@ private void doHandleDelete(String pid) throws IOException {
         Files.delete(fileFromCache.toPath());
         SecurityLogger.audit("Removed a deleted config [{}]", fileFromCache.getAbsolutePath());
       } catch (IOException e) {
-        LOGGER.debug(
```

Comment:
Are you sure this isn't the correct way to do this with a throwable? The signature on logger.debug(String,Object...) does it really treat e like a throwable and print the stack trace?

Reply:
never-mind this has been fixed since slf4j 1.6.0 https://www.slf4j.org/faq.html#paramException